### DISCUSSION TITLE
How to run compound expression? Is my query valid??

### DISCUSSION THREAD

**Discussion by Rupesh253** [nunit/nunit#4459] [How to]
Hi Team, 

I have 5 testclasses. Class1, Class2, Class3, Class4, Class5 with categories as Smoke

Now, i need to execute cat==Smoke and not class1 and class2. 

Tried out this expression: **_nunit3-console "testproject.dll" --where "cat==smoke && ((class!=Class1)||(class!=Class2))"_** 

but it executed all tests based on cat==smoke

**Comment by manfred-brands** [comment]
You need to && between the class condition.

In your case every class matches.
Class1 => false || true
Class2 => true || false
